qiskit/transpiler/synthesis/aqc/aqc.py

```diff
@@ -96,17 +161,26 @@ def compile_unitary(
         # set the matrix to approximate in the algorithm
         approximating_objective.target_matrix = su_matrix
 
-        optimizer = self._optimizer or L_BFGS_B(maxiter=1000)
+        optimizer = self._optimizer or partial(
```

Comment:
Do you know if there's a reason why this is set upon `compile_unitary` and not just in the initializer?

Reply:
I don't know, I think it could be moved to the initializer without issues.